README.md

```diff
@@ -84,7 +84,7 @@ ARGUMENTS
 FLAGS
   -r, --remote=<value>      git remote of target app
   -s, --source-app=<value>  (required) source app for model resource
-  -t, --target-app=<value>  (required) target app for model resource
+  -t, --target-app=<value>  (required) [default: heroku-vscode] target app for model resource
```

Comment:
Please remove all the places where `[default: heroku-vscode]` has been added to the documentation. This happens because an app has been set for the `heroku` origin in your local .git/config file in the repository.

Reply:
my favorite oclif feature and the one that makes the most sense to have in there!